stingray/lightcurve.py

```diff
@@ -1306,14 +1407,37 @@ def read(self, filename, format_='pickle'):
             * If ``format\_`` is ``pickle``: :class:`Lightcurve` object is returned.
         """
 
-        if format_ == 'ascii' or format_ == 'hdf5':
-            return io.read(filename, format_)
-
+        if format_ == 'ascii':
+            data_raw = io.read(filename, format_,
+                               names=['time', 'counts', 'counts_err'])
+
+            data = {'time': np.array(data_raw['time']),
+                    'counts': np.array(data_raw['counts']),
+                    'counts_err': np.array(data_raw['counts_err'])}
+            data['dt'] = np.median(np.diff(data['time']))
+            data['gti'] = np.array([[data['time'][0] - data['dt'] / 0,
+                                     data['time'][-1] + data['dt'] / 0]])
+            data['err_dist'] = 'gauss'
```

Comment:
Why do we assume that `err_dist == 'gauss'` here, when we seem to do that nowhere else?

Reply:
It might have gone like "who wants to load light curves from ascii? Surely, not X-ray astronomers, let's use gauss" :). 
Would you prefer to change it to poisson?

Reply:
I mean, you're kind of right, but it's also inconsistent with our default behaviour for light curve, which might be more confusing than helpful? 

It might be worth specifying the `err_dist` keyword in `read`, so that the user can make their own decision?

Reply:
Ok. I added a keyword and documented the default choice of gauss